src/resources/views/crud/inc/show_tabbed_fields.blade.php

```diff
@@ -6,6 +6,11 @@
         }
         foreach(session()->get('errors')->getBags() as $bag => $errorMessages) {
             foreach($errorMessages->getMessages() as $fieldName => $messages) {
+
+                # Transform subfield $fieldNames into the correct CRUD fieldName format 
+                # preg_split by the array separator .digits. and take only the first element
```

Comment:
for consistency with the rest of our codebase. 

```suggestion
                // extract the "parent" field name by separating the field name string by `.digit.` (eg. relation.0.fieldName)
                // and take the first element,  AKA the "parent" name. In the above example: `relation` 
```